packages/cli/src/commands/deploy.ts

```diff
@@ -83,16 +83,17 @@ export default class Deploy extends Command {
 
       const { data } = await api.projects.deploy(projectPayload, { dryRun: preview })
       if (preview || output) {
-        console.info(data)
+        this.log(data)
       }
       if (!preview) {
-        console.info(`Successfully deployed project "${project.name}" to account "${account.name}".`)
+        this.log(`Successfully deployed project "${project.name}" to account "${account.name}".`)
       }
     } catch (err: any) {
       if (err?.response?.status === 400) {
-        console.error(`Failed to deploy the project due to a missing field. ${err.response.data.message}`)
+        throw new Error(`Failed to deploy the project due to a missing field. ${err.message}`)
```

Comment:
I know that I mentioned we shouldn't use `err.response.data.message`. In this case, though, (when there's a `400` response code) - I think that it's OK to leave the error as it was before.

```suggestion
        throw new Error(`Failed to deploy the project due to a missing field. ${err.response.data.message}`)
```